src/CommerceModel.BetterRetail/artifacts/OOE/BetterRetail/QueueImportProfileSchemaActivity/profiles.json

```
@@ -296,65 +279,139 @@
         }
       },
       {
-        "groupId": "Default",
+        "groupId": "AttributesGroupTest",
+        "isRequired": false,
+        "displayOrder": 0,
+        "dataType": "Boolean",
+        "isBuiltIn": false,
+        "allowMultipleValues": false,
+        "multilingual": false,
+        "referenceTypeName": "TestBoolean",
+        "isSearchable": false,
+        "name": "TestBoolean",
+        "displayName": {
+          "en-US": "Test Boolean",
+          "fr-CA": "Test boolean",
+          "fr-FR": "Test boolean"
+        }
+      },
+      {
+        "groupId": "AttributesGroupTest",
+        "isRequired": false,
+        "displayOrder": 0,
+        "dataType": "DateTime",
+        "isBuiltIn": false,
+        "allowMultipleValues": false,
+        "multilingual": false,
+        "referenceTypeName": "TestDatetime",
+        "isSearchable": false,
+        "name": "TestDatetime",
+        "displayName": {
+          "en-US": "Test DateTime",
+          "fr-CA": "Test date",
+          "fr-FR": "Test date"
+        }
+      },
+      {
+        "groupId": "AttributesGroupTest",
+        "isRequired": false,
+        "displayOrder": 0,
+        "dataType": "Decimal",
+        "isBuiltIn": false,
+        "allowMultipleValues": false,
+        "multilingual": false,
+        "referenceTypeName": "TestDecimal",
+        "isSearchable": false,
+        "name": "TestDecimal",
+        "displayName": {
+          "en-US": "Test Decimal",
+          "fr-CA": "Test d�cimale",
+          "fr-FR": "Test d�cimale"
+        }
+      },
+      {
+        "groupId": "AttributesGroupTest",
+        "isRequired": false,
+        "displayOrder": 0,
+        "dataType": "Text",
+        "isBuiltIn": false,
+        "allowMultipleValues": false,
+        "multilingual": false,
+        "referenceTypeName": "TestText",
+        "isSearchable": false,
+        "name": "TestText",
+        "displayName": {
+          "en-US": "Test Text",
+          "fr-CA": "Test Texte",
+          "fr-FR": "Test Texte"
+        }
+      },
+      {
+        "groupId": "AttributesGroupTest",
+        "isRequired": false,
+        "displayOrder": 0,
+        "dataType": "Integer",
+        "isBuiltIn": false,
+        "allowMultipleValues": false,
+        "multilingual": false,
+        "referenceTypeName": "TestInteger",
+        "isSearchable": false,
+        "name": "TestInteger",
+        "displayName": {
+          "en-US": "Test Integer",
+          "fr-CA": "Test Integer",
+          "fr-FR": "Test Integer"
```

I think it's "entier", not sure
```suggestion
          "fr-CA": "Test entier",
          "fr-FR": "Test entier"
```